appinventor/components/src/com/google/appinventor/components/common/YaVersion.java

```diff
@@ -499,8 +499,10 @@ private YaVersion() {
   // - FORM_COMPONENT_VERSION was incremented to 27.
   // For YOUNG_ANDROID_VERSION 201:
   // - CANVAS_COMPONENT_VERSION was incremented to 13
+  // For YOUNG_ANDROID_VERSION 202:
+  // - BLOCKS_LANGUAGE_VERSION was incremented to 30
```

Comment:
I would recommend against bumping YOUNG_ANDROID_VERSION since there's no guarantee that this will be the next PR to be merged that affects YOUNG_ANDROID_VERSION. We will push a commit to your branch when we are ready to merge it to bump the version number here.